client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -123,6 +123,7 @@ const useInteractionsTab = (parameters: useInteractionsTabParameters): useIntera
                     params: {contactEventId}
                 }).then(() => {
                     setInteractions(interactions.filter((interaction: InteractionEventDialogData) => interaction.id !== contactEventId));
+                    loadInteractions();
```

Comment:
There is no need in querying again all interactions, just do setAreThereContacts

Reply:
after deleting you need to check the new number of contacts and then after reducing them you can set the areThereContacts state according to the new result.